src/Stripe.Tests.XUnit/Stripe.Tests.XUnit.csproj

```diff
@@ -1,14 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFrameworks>net452;netcoreapp1.0;netcoreapp1.1;</TargetFrameworks>
```

Comment:
this little "s" at the end might have been why you had trouble. It tricked me for a bit.